packages/contracts-bedrock/deploy-config/goerli-forked.json

```diff
@@ -0,0 +1,29 @@
+{
+  "finalSystemOwner": "0xf80267194936da1E98dB10bcE06F3147D580a62e",
+  "controller": "0xf80267194936da1E98dB10bcE06F3147D580a62e",
+
+  "l1StartingBlockTag": "latest",
+  "l1ChainID": 5,
+  "l2ChainID": 111,
+  "l2BlockTime": 2,
+
+  "maxSequencerDrift": 1000,
+  "sequencerWindowSize": 120,
+  "channelTimeout": 120,
+  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000002",
+  "batchSenderAddress": "0xa11d2b908470e17923fff184d48269bebbd9b2a5",
+
+  "systemConfigOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
+
+  "l2OutputOracleSubmissionInterval": 6,
+  "l2OutputOracleStartingTimestamp": -1,
```

Comment:
THis is a uint256, how does `-1` work here?

Reply:
Special value, -1 is treated as "does not exist" but fwiw I don't even think this is being used directly anymore